src/appkit.cs

```diff
@@ -26692,4 +26724,138 @@ interface NSWindowTabGroup
 		[Export ("removeWindow:")]
 		void Remove (NSWindow window);
 	}
+
+	[Mac (10,15)]
+	[Native]
+	public enum NSTextScalingType : long
+	{
+		Standard = 0,
+		iOS,
+	}
+
+	[Mac (10,15)]
+	[BaseType (typeof(NSTouchBarItem))]
+	[DisableDefaultCtor]
+	interface NSButtonTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Static]
+		[Export ("buttonTouchBarItemWithIdentifier:title:target:action:")]
+		NSButtonTouchBarItem Create (NSTouchBarItemIdentifier identifier, string title, [NullAllowed] NSObject target, [NullAllowed] Selector action);
```

Comment:
Using `NSTouchBarItemIdentifier` is much better but it's not the right syntax.

The API accept `NSString` not an `enum`. To use our _smart _ (field-backed) `enum` you need to use `[BindAs]`, e.g.

```
NSButtonTouchBarItem Create ([BindAs (typeof (NSTouchBarItemIdentifier))] NSString identifier, ...
```

This will instruct the generator to add the conversion logic (from `enum` to `NSString`) in the generated code.

You can look it up by finding the file `NSButtonTouchBarItem.g.cs` (under `/src/build/`) and see how it differs from what you have right now.